docs/docs/intro.md

```diff
@@ -12,6 +12,6 @@ beyond M365, will expand based on the interest and needs of the community.
 
 ## Getting started
 
-You can follow the [Quick Start](quickstart) guide for an end-to-end Corso walk through. Alternatively, follow
-the instructions in the [Corso Setup](setup/concepts) section to dive into the details on how to configure and
+You can follow the [Quick Start](../quickstart) guide for an end-to-end Corso walk through. Alternatively, follow
+the instructions in the [Corso Setup](../setup/concepts) section to dive into the details on how to configure and
```

Comment:
@ntolia should this have the trailing slash as above?

Reply:
Fixed the above issue but it's a no-op with this setting.